webrtc.html

```diff
@@ -4685,7 +4685,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         <li>
           <p>If the <a href="#dfn-underlying-data-transport">transport</a> was
           closed <dfn id="data-transport-closed-error">with an error</dfn>,
-          fire an NetworkError event at <var>channel</var>.</p>
+          queue a task to fire an NetworkError event at <var>channel</var>.</p>
```

Comment:
This one is a bit strange. Is the "NetworkError event" even defined somewhere? I think we could do with one event here.